shared/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
   alias(libs.plugins.androidmakers.kmp.library)
+  alias(libs.plugins.moko)
```

Comment:
Again, more of a general comment, definitely not a blocker but we might want to start making more shared modules. Putting resources together with networking and cache feels a bit "too many responsibilities"

Reply:
Okay, I should have commented this one.

For iOS to work with KMP, we need to expose a single module. That's why I added a "shared" module over the existing business modules. 

TMO, Moko is a temporary solution. It should be removed if we manage to implement Compose Multiplatform. The thing is, the resources files are used almost the same way with Moko and Compose MP. So I chose to use Moko for the moment to start sharing resources and be able to move quickly to Compose in short term.

Reply:
Makes a ton of sense 👍 Thanks for the explanation 👍